### PR TITLE
Create backup and restore section for 389 Directory Server in the Sec…

### PR DESCRIPTION
**Pull request by cjschroder**
…urity Guide

Jira DOCTEAM-313

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/security_ldap_backups.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    <command>dsctl</command> command makes offline database backups, and the
+    <command>dsconf</command> makes online database backups. Back up the LDAP server configuration directory, to enable complete restoration in case of a major failure.
+  </para>
+  
+  <sect2 xml:id="sec-security-ldap-backup-config">
+    <title>Backing Up the Server Configuration</title>
+      <para>
+        Your LDAP server configuration is in 
+        <filename>/etc/dirsrv/slapd-<replaceable>instance-name</replaceable></filename>. This contains certificates, keys, and the
+        RootDSE file. Make a compressed backup of this directory with the
```
Comment (Firstyear) [inline xml/security_ldap_backups.xml:25]: rootdse -> dse.ldif configuration

**Review by Firstyear — APPROVED** [review]
👍🎉

File: xml/security_ldap_backups.xml (hunk 1 above)
Comment (cjschroder) [reply xml/security_ldap_backups.xml]: ok!

**Comment by cjschroder** [comment]
I adore the concise commands. so much better than typing out paragraphs.

**Comment by cjschroder** [comment]
SLE 15 SP3: 578931e37

**Pull request merged**